python/paddle/fluid/tests/unittests/test_lrn_op.py

```diff
@@ -154,5 +155,207 @@ def test_errors(self):
+                x=input2, size=5, data_format='NLC')
+            exe = fluid.Executor(place)
+            fetches = exe.run(fluid.default_main_program(),
+                              feed={"input1": in_np1,
+                                    "input2": in_np2},
+                              fetch_list=[res1, res2])
+
+            fetches1_tran = np.transpose(fetches[1], (0, 2, 1))
+            self.assertTrue(np.allclose(fetches[0], fetches1_tran))
+
+    def check_static_4d_input(self, place):
+        with fluid.program_guard(fluid.Program(), fluid.Program()):
+            input1 = fluid.data(
+                name="input1", shape=[3, 3, 40, 40], dtype="float32")
+            input2 = fluid.data(
+                name="input2", shape=[3, 40, 40, 3], dtype="float32")
+
+            res1 = paddle.nn.functional.local_response_norm(
+                x=input1, size=5, data_format='NCHW')
+            res2 = paddle.nn.functional.local_response_norm(
+                x=input2, size=5, data_format='NHWC')
+
+            in_np1 = np.random.random([3, 3, 40, 40]).astype("float32")
+            in_np2 = np.transpose(in_np1, (0, 2, 3, 1))
+
+            exe = fluid.Executor(place)
+            fetches = exe.run(fluid.default_main_program(),
+                              feed={"input1": in_np1,
+                                    "input2": in_np2},
+                              fetch_list=[res1, res2])
+
+            fetches1_tran = np.transpose(fetches[1], (0, 3, 1, 2))
+            self.assertTrue(np.allclose(fetches[0], fetches1_tran))
+
+    def check_static_5d_input(self, place):
+        with fluid.program_guard(fluid.Program(), fluid.Program()):
+            input1 = fluid.data(
+                name="input1", shape=[3, 3, 3, 40, 40], dtype="float32")
+            input2 = fluid.data(
+                name="input2", shape=[3, 3, 40, 40, 3], dtype="float32")
+            res1 = paddle.nn.functional.local_response_norm(
+                x=input1, size=5, data_format='NCDHW')
+            res2 = paddle.nn.functional.local_response_norm(
+                x=input2, size=5, data_format='NDHWC')
+
+            in_np1 = np.random.random([3, 3, 3, 40, 40]).astype("float32")
+            in_np2 = np.transpose(in_np1, (0, 2, 3, 4, 1))
+
+            exe = fluid.Executor(place)
+            fetches = exe.run(fluid.default_main_program(),
+                              feed={"input1": in_np1,
+                                    "input2": in_np2},
+                              fetch_list=[res1, res2])
+
+            fetches1_tran = np.transpose(fetches[1], (0, 4, 1, 2, 3))
+            self.assertTrue(np.allclose(fetches[0], fetches1_tran))
+
+    def test_static(self):
+        for place in self.places:
+            self.check_static_3d_input(place=place)
+            self.check_static_4d_input(place=place)
+            self.check_static_5d_input(place=place)
+
+    def check_dygraph_3d_input(self, place):
```

Comment:
so it only check the data_format?

Reply:
Yes, Acc is aligned with torch implementation for simplicity,  here we just test data_format